packages/edit-site/src/components/global-styles/screen-revisions/use-global-styles-revisions.js

```diff
@@ -32,14 +32,22 @@ export default function useGlobalStylesRevisions() {
 			__experimentalGetDirtyEntityRecords,
 			getCurrentUser,
 			getUsers,
-			getCurrentThemeGlobalStylesRevisions,
+			getRevisions,
+			__experimentalGetCurrentGlobalStylesId,
 			isResolving,
 		} = select( coreStore );
 		const dirtyEntityRecords = __experimentalGetDirtyEntityRecords();
 		const _currentUser = getCurrentUser();
 		const _isDirty = dirtyEntityRecords.length > 0;
 		const globalStylesRevisions =
-			getCurrentThemeGlobalStylesRevisions() || EMPTY_ARRAY;
+			getRevisions(
+				'root',
+				'globalStyles',
+				__experimentalGetCurrentGlobalStylesId(),
+				{
+					per_page: 100,
```

Comment:
At the moment there's a limit of 100 revisions set by [WP_REST_Controller](https://github.com/ramonjd/wordpress-develop/blob/2a0d8fcd645e0df939eed588b1cc832e568a1150/src/wp-includes/rest-api/endpoints/class-wp-rest-controller.php#L357-L357)

Limit the results per page to 100 until we can implement pagination. See:

- https://github.com/WordPress/gutenberg/issues/53621

Thanks to @andrewserong for reminding me 😴